onnxruntime/test/providers/cpu/nn/conv_integer_test.cc

```diff
@@ -92,6 +132,48 @@ TEST(ConvIntegerTest, WithGroup_2D_u8u8) {
   test.Run();
 }
 
+TEST(ConvIntegerTest, WithGroup_2D_u8s8) {
+  OpTester test("ConvInteger", 10);
+  std::vector<int64_t> x_dims{1, 3, 3, 3};
+  test.AddInput<uint8_t>("x", x_dims,
+                         {2, 3, 4,
+                          5, 6, 7,
+                          8, 9, 10,
+                          11, 12, 13,
+                          14, 15, 16,
+                          17, 18, 19,
+                          20, 21, 22,
+                          23, 24, 25,
+                          26, 27, 28});
+  std::vector<int64_t> w_dims{3, 1, 2, 2};
+  test.AddInput<int8_t>("w", w_dims,
```

Comment:
could you please add both positive and negative weight to cover more cases?

Reply:
Sure